staff_features/shared/step_definitions.rb

```diff
@@ -83,6 +84,12 @@
   end
 end
 
+When 'the user clicks on {string} in the confirm modal' do |string|
```

Comment:
Maybe just an artifact of too many PRs queued, but this function (or one very similar) was removed in PR #38 and its being added back here, but doesn't seem to be called in this PR

Reply:
Yes it has been removed somewhere on previous requests, but not from here. Removal was going to be applied on merge/rebase.